### PR TITLE
docs: fleek functions platform

### PR DESCRIPTION
**Pull request by tobySolutions**
## Why?

Update the Fleek functions platform docs warning to reflect new information and to be shorter with the latest information on Fleek Functions.

## Contribution checklist?

- [x] The commit messages are detailed
- [x] The `build` command runs locally
- [x] Assets or static content are linked and stored in the project
- [x] Document filename is named after the slug
- [x] You've reviewed spelling using a grammar checker
- [x] For documentation, guides or references, you've tested the commands and steps
- [x] You've done enough research before writing

### Before

Screenshot 2024-08-26 at 13 54 29

### After

Screenshot 2024-08-26 at 13 54 34

### CODE REVIEW

File: src/layouts/DocPage.astro [hunk 1]
```diff
@@ -78,7 +78,7 @@ const indexNameDocs = import.meta.env.PUBLIC_MEILISEARCH_INDEX_DOCS;
 const isHome = Astro.url.pathname === '/docs';
 
 const slug = `docs/${data.slug}`;
-const image = '';
+const image = settings.site.metadata.docs.image;
```
Comment (davidlfox) [inline src/layouts/DocPage.astro]: does this somehow translate to the other values e.g. `settings.site.metadata.guides.image` in guides.astro?

**Comment by davidlfox** [comment]
@tobySolutions re: failed staging build, see our conversation in #dev-rel. something going on there that i dont fully understand

 @theBeardA @heldrida

**Comment by tobySolutions** [comment]
> @tobySolutions re: failed staging build, see our conversation in #dev-rel. something going on there that i dont fully understand
> 
> @theBeardA @heldrida

Some changes there were clashing in. I removed them now.

**Review by davidlfox — APPROVED** [review]
looks good. i know the rest of the page sort of explains what our "roadmap" is, so this can approve for now!

**Pull request merged**